### PR TITLE
Update Black dependency to fix CI lint

### PR DESCRIPTION
**Pull request by intgr**
Closes #896.

Black 22.3.0 fixes incompatibility with Click 8.1.0: https://github.com/psf/black/pull/2966

This currently causes the CI to fail, e.g. https://github.com/typeddjango/django-stubs/runs/5756075543

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ repos:
         types: [pyi]
         args: ["--profile", "black"]
   - repo: https://github.com/psf/black
-    rev: 21.12b0
+    rev: 22.3.0
```
Comment (intgr) [inline .pre-commit-config.yaml:31]: Looks like `.pre-commit-config.yaml` hasn't been kept in sync with requirements.txt as it should be.

**Comment by intgr** [comment]
I'm not sure why it's still failing, but it's a different message this time.

**Comment by sobolevn** [comment]
You need to reformat some files with new black version.

**Comment by intgr** [comment]
Ahh! Makes sense, I will do it tomorrow.

**Comment by intgr** [comment]
Fixed, should be ready to merge now.

**Review by sobolevn — APPROVED** [review]

**Comment by sobolevn** [comment]
Thank you!

**Pull request merged**